CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 - PostgreSQL working copy no longer requires PostGIS for aspatial datasets. [#729] (https://github.com/koordinates/kart/issues/729)
 - Fixes a bug where the current spatial filter isn't stored in the filesystem working copy, affecting the spatial filtering of point cloud datasets. [#833](https://github.com/koordinates/kart/pull/833)
 - Fixes pthread_key leaks in a long running Kart process due to repeated loading and unloading of mod_spatialite. [#838](https://github.com/koordinates/kart/pull/838)
+- Fixes a bug where features are written to the working copy without their SRID during `kart resolve`. [#840](https://github.com/koordinates/kart/issues/840)
```

Comment:
Use 'CRS identifier' rather than SRID - SRID is a PostGIS-specific term

Reply:
Done

Reply:
Done